migration/migration/004-schemas.sql

```diff
@@ -47,7 +51,7 @@ DO $$
 DECLARE
    new_path text;
 BEGIN
-   new_path := current_setting('search_path') || format(',%L,%L,%L,%L,%L,%L', 'ps_tag', '_prom_ext', 'prom_api', 'prom_metric', '_prom_catalog', 'ps_trace');
+   new_path := format('%L,%L,%L,%L,%L,%L,%L,%L', '$user','public', 'ps_tag', '_prom_ext', 'prom_api', 'prom_metric', '_prom_catalog', 'ps_trace');
```

Comment:
`format` is not strictly necessary here anymore, no? It's just one big literal.

Reply:
I think format is more safe and future-proof so I'd rather leave. This code will go away in 166 anyway